src/utils/hooks/useGetQuestionReviewHistory.tsx

```diff
@@ -55,18 +53,18 @@ const useGetQuestionReviewHistory = ({ isApplicant, ...variables }: UseGetQuesti
       const { firstName, lastName } = application?.user as User
 
       if (stageNumber) {
-        if (!allResponsesByStage[stageNumber]) allResponsesByStage[stageNumber] = {}
-        // Set each entry using HistoryElement values
-        allResponsesByStage[stageNumber][timeUpdated] = {
+        if (!allResponsesByStage[stageNumber]) allResponsesByStage[stageNumber] = []
+
+        allResponsesByStage[stageNumber].push({
```

Comment:
Easier to make this an array of responses rather than object indexed by timestamp (which can be identical!)